### PR TITLE
Hello world program edits

### PR DESCRIPTION
**Pull request by DheerajBeerelly**
changed hello world to swapping of two integers

### CODE REVIEW

File: HelloWorldProgram.c [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+ 
+int main()
+{
+  int x, y, t;
```
Comment (harinarayana1997) [inline HelloWorldProgram.c]: Use some meaning full variable names rather than a, b, c, x, z or something.

File: hello-world-program.c [hunk 2]
```diff
@@ -1,5 +1,19 @@
-#include<stdio.h>
-void main()
+#include <stdio.h>
+ 
+int main()
 {
-  printf("Hello World");
+  int x, y, t;
```
Comment (Abhivardhan) [inline hello-world-program.c]: Its fine for now but try to name the variables properly whether its for practice or a serious project.

File: HelloWorldProgram.c [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ 
+  printf("Enter two integers\n");
+  scanf("%d%d", &x, &y);
+  printf("Before Swapping\nFirst integer = %d\nSecond integer = %d\n", x, y);
+  swap(&x,&y); 
+  printf("After Swapping\nFirst integer = %d\nSecond integer = %d\n", x, y);
+  return 0;
+}
+
+void swap(int *a, int *b)
```
Comment (harinarayana1997) [inline HelloWorldProgram.c]: The main method should be the last function. Write all the functions above the main function.

Comment (harinarayana1997) [reply HelloWorldProgram.c]: And this program won't run as there is no function declaration at the top. To avoid it write all the functions before main.

**Pull request merged**